util/chplenv/chpl_lib_pic.py

```diff
@@ -9,7 +10,20 @@
 def get():
     lib_pic_val = overrides.get('CHPL_LIB_PIC')
     if not lib_pic_val:
-        lib_pic_val = 'none'
+        # Some platforms have 'gcc' and/or standard libraries
+        # default to PIC. For those platforms, arrange for CHPL_LIB_PIC
+        # to defaurt to 'pic' instead of 'none' to avoid problems.
+        #
+        # Another option would be to check for --enable-default-pie in
+        #   gcc -v -E
+        # However, that would assume that 'gcc' is related to the link command,
+        # which is not necessarily the case.
+        release = platform.uname()[2]
+        if 'arch' in release:
```

Comment:
My only concern with this route is if there are distributions that include, say, "March" in them in some way (maybe as part of distinguishing when a release occurred) or "Architecture" or some other word that contains "arch".  What exactly does Arch Linux display?

Reply:
e.g. '5.17.5-arch1-1'. 

An alternative strategy would be to check if `/etc/arch-release` exists

Reply:
I feel better about checking that, that seems less likely to result in false positives (and any false negatives can be added to the check when they are discovered)